src/main/java/com/company/KafkaProducerTest.java

```diff
@@ -45,11 +45,15 @@ public static void main(String[] args) {
 //    for (int i = 0; i < 100; i++) {
```

Comment:
thanks for the comment, the line has been deleted and the change has been pushed to the logreader